src/main/java/im/status/keycard/SecureChannel.java

```diff
@@ -176,6 +199,66 @@ public void mutuallyAuthenticate(APDU apdu) {
     respond(apdu, len, ISO7816.SW_NO_ERROR);
   }
 
+  /**
+   * Processes the LOAD_CERTS command. Copies the APDU buffer into `certs`.
+   * This function expects a DER signature and may only be called once.
+   * @param apdu the JCRE-owned APDU object.
+   */
+  public void loadCert(APDU apdu) {
+    byte[] apduBuffer = apdu.getBuffer();
+
+    final short caPubOffset = (short) ISO7816.OFFSET_CDATA;
+    final short certOffset = (short) (caPubOffset + PUBKEY_LEN);
+    final short certVerifyLen = (short)(CERTIFICATE_LEN - ECDSA_MAX_LEN);
```

Comment:
This variable is never used

Reply:
Out of date comment. This was cleaned up a few days ago